app/models/translation.rb

```diff
@@ -10,6 +10,22 @@ class Translation < ActiveRecord::Base
 
   @@root_folder_name = 'public/srt/'
 
+  @@INCOMPLETE_STATUS = "incomplete"
+  @@COMPLETE_STATUS = "complete"
+  @@COMPLETE_WITH_PRIORITY_STATUS = "priority"
+
+  @@COMPLETE_POINTS = 1
+  @@COMPLETE_WITH_PRIORITY_POINTS = 2
+
+  def points
+    if translation.status == @@COMPLETE_STATUS
+        return @@COMPLETE_POINTS
```

Comment:
You editor suddenly interpreted an indent as 4 spaces. You should change it back.